src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -358,6 +358,12 @@ void handle_convert_worker_to_actor(
      * filled out, so fill out the correct worker field now. */
     algorithm_state->local_actor_infos[actor_id].worker = worker;
   }
+  /* Increment the task counter for the creator's handle to account for the
+   * actor creation task. */
+  auto &task_counters =
+      algorithm_state->local_actor_infos[actor_id].task_counters;
+  RAY_CHECK(task_counters[ActorHandleID::nil()] == 0);
+  task_counters[ActorHandleID::nil()]++;
```

Comment:
Can you explain this change? I wouldn't have expected any changes to the old code path.

Reply:
It's because I now count the actor creation task as part of the actor counter on the Python side. Without this change, the old codepath will never run any tasks after the actor creation task since the counters don't match.